jaxrs2/src/main/java/io/opentracing/contrib/jaxrs/client/ClientTracingFeature.java

```diff
@@ -0,0 +1,106 @@
+package io.opentracing.contrib.jaxrs.client;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.ws.rs.client.Client;
+
+import io.opentracing.Tracer;
+
+/**
+ * @author Pavol Loffay
+ */
+public class ClientTracingFeature {
+
+    private static final Logger log = Logger.getLogger(ClientTracingFeature.class.getName());
+
+    private ClientTracingFeature(Builder builder) {
+        if (log.isLoggable(Level.INFO)) {
+            log.info("Registering client tracing for: " + builder.client);
+        }
+
+        builder.client.register(new SpanClientRequestFilter(builder.tracer, builder.spanDecorators), 0)
+                .register(new SpanClientResponseFilter(builder.spanDecorators), 0);
+    }
+
+    /**
+     * Builder for configuring {@link Client} to trace outgoing requests.
+     *
+     * By default span operation name is set by {@link ClientSpanDecorator#HTTP_METHOD_OPERATION_NAME} and
+     * span is decorated with {@link ClientSpanDecorator#STANDARD_TAGS}.
+     */
+    public static class Builder {
+        private Tracer tracer;
+        private Client client;
+        private List<ClientSpanDecorator> spanDecorators = new ArrayList<>();
```

Comment:
Builder has two methods

`withEmptyDecorators()` which clears array and `withDecorator(decorator)` which adds decorator.
By default in a constructor it adds two decorators one for operation name and other for standard tags.

It makes sense to provide only `Builder#withDecorators(decorators)` I think it's cleaner :+1: